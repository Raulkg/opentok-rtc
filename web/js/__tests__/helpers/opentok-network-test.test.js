var ont = require('../../helpers/opentok-network-test.js');
test('snapshot for opentok-network-test component', () => {
  expect(ont).toMatchSnapshot();
});