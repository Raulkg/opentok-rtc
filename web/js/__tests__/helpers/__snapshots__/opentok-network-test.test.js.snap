// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for opentok-network-test component 1`] = `
Object {
  "OTNetworkTest": [Function],
}
`;
